src/xcode/ENA/ENAUITests/AppStore/ENAUITests.swift

```diff
@@ -133,4 +121,18 @@ class ENAUITests: XCTestCase {
 		if snapshotsActive { snapshot("AppStore_0006") }
 	}
 
+	func test_0002_Generate_Screenshot_For_AppStore_Statistics() throws {
+
+		app.setPreferredContentSizeCategory(accessibility: .normal, size: .M)
+		app.launchArguments.append(contentsOf: ["-isOnboarded", "YES"])
+		app.launchArguments.append(contentsOf: ["-ENStatus", ENStatus.active.stringValue])
+		app.launchArguments.append(contentsOf: ["-useMockDataForStatistics", "YES"]) // prevent failing tests for 1.11; use "NO" for 1.12
```

Comment:
As we are in `2.x` already shouldn't `useMockDataForStatistics` be  `NO` then?
On the other hand I would be fine with mock stats for faster and more reliable testing.